### PR TITLE
6711 - Added CSS classes for IDS Status Colors to target color and border-color properties

### LINKED ISSUE #6711: Color Palette (color.scss): Create IDS status color classes targetting color, border-color CSS properties
**Is your feature request related to a problem or use case? Please describe.**
I am creating a new workbrain-lib component which uses border-color for linking items, color for link icons (SVGs). We'd like the color choices to be limited to the five IDS status colors.

**Describe the solution you'd like**
Add classes similar to '.status-01' that target the color and border-color properties instead. Something like the following is perfect:
`.status-01-border {
    border-color: $status-01-bg-color;
} 
.status-01-color {
    color: $status-01-bg-color;
}`

**Describe alternatives you've considered**
Currently I have internal CSS classes in the angular components that assign the color/border-color properties with the $status-01-bg-color variable used in the status-colors in the color.scss file. An IDS class would be nicer for consistency throughout the application.

**Additional context**
Application: Workforce Management - Portal
Team: curiosity (Jacob)
Related internal ticket: WFM-44554, WFM-46441

### PR DESCRIPTION
**Pull request by royaljuggy**
Resolves #6711 - Added CSS classes for IDS Status Colors affecting color and border-color properties

Added '.status-XX-border' and '.status-XX-color' classes in the colors.scss that apply status colors to border-color and color properties respectively
Renamed 'status-XX-bg-color' variables in config.scss to a more general 'status-XX-color' name
Included proposed changes in CHANGELOG

**Explain the _details_ for making this change. What existing problem does the pull request solve?**
These new classes allow developers to apply the IDS status colors to border color and color CSS properties (not just background colors)

**Related github/jira issue (required)**:
Resolves #6711
Resolves / Enhances WFM-44554
Resolves / Enhances WFM-46441

**Steps necessary to review your pull request (required)**:
Test if we can actually apply these classes (for borders -> divs, for colors -> IDS icons or text)

**Included in this Pull Request**:
- A note to the change log.

### CODE REVIEW

**Comment by tmcconechy** [comment]
@royaljuggy thanks looks good. maybe for a test you could add something to the color page below the status colors http://localhost:4000/components/colors/example-index.html

Maybe something like this that uses the classes

Screen Shot 2022-08-04 at 2 07 44 PM

Demo css in the css page. But i have no problems with this addition. But wanted to mention the fields all have classes https://main-enterprise.demo.design.infor.com/components/validation/test-alert-types.html already

**Comment by royaljuggy** [comment]
> @royaljuggy thanks looks good. maybe for a test you could add something to the color page below the status colors http://localhost:4000/components/colors/example-index.html
> 
> Maybe something like this that uses the classes
> 
> Screen Shot 2022-08-04 at 2 07 44 PM
> 
> Demo css in the css page. But i have no problems with this addition. But wanted to mention the fields all have classes https://main-enterprise.demo.design.infor.com/components/validation/test-alert-types.html already

Ooo, the existing classes could definitely be used. Though, in my opinion it might be easier for users to also have the "status-XX" classes, since you can easily view their colors at https://design.infor.com/code/ids-enterprise/latest/colors - let's move with this change after I get the demo css page updated, and the automated builds passing

**Comment by ericangeles** [comment]
Approving to run the build

File: docs/CHANGELOG.md [hunk 1]
```diff
@@ -12,7 +12,7 @@
 - `[Searchfield]` Added option to add custom icon button. ([#6453](https://github.com/infor-design/enterprise/issues/6453))
 - `[Targeted-Achievement]` Added puppeteer test for show tooltip on targeted achievement. ([#6550](https://github.com/infor-design/enterprise/issues/6550))
 - `[Textarea]` Converted protractor tests to puppeteer. ([#6629](https://github.com/infor-design/enterprise/issues/6629))
-
+- `[Color Palette]` Added status color CSS classes for color and border-color properties. ([#6711](https://github.com/infor-design/enterprise/issues/6711))
```
Comment (tmcconechy) [inline docs/CHANGELOG.md:15]: @royaljuggy can you merge main on and add this to a section in 4.67.0 fixes instead.

We are on 4.67 now. Also should be in alpha order there. This is the why there is a lint error.

Rest looks good to me.

Comment (royaljuggy) [reply docs/CHANGELOG.md]: Rebased/merged new commits from enterprise/main into my branch
- added demo views to http://localhost:4000/components/colors/example-index.html
- updated CHANGELOG as per comments

**Review by ericangeles — APPROVED** [review]

**Pull request merged**